Dockerfile.amd64

```diff
@@ -12,7 +12,14 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM registry.access.redhat.com/ubi8/ubi-minimal:8.1-407
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.2-301.1592810506 AS base
+
+RUN mkdir /licenses
+ADD licenses/ /licenses
+ADD LICENSE /licenses/
+RUN mkdir -p /opt/cni/bin
```

Comment:
Couple of questions:

- Do we need the mkdir `/opt/cni/bin` - we don't have this for the other dockerfiles below, so seems odd that we would need it here.  If we do need it here, could we just add the binary to / instead?
-  I'm still trying to understand why we need the Ubi base image.  AFAICT it's really just for the license. Is that correct?  Is there something special about that copy of the license file?

Couldn't we rationalize all of these docker files to be from scratch, copy in a license file from the local file system and then just copy across the appropriate build of the binary.  That way all of the docker files look pretty similar.

Reply:
We do need to create /opt/cni/bin. The copy later on in scratch (ADD bin/amd64/ /opt/cni/bin/) will fail otherwise.
Regarding UBI, I am not the best to answer. It was there and I left it there.

Reply:
While you're here, could you please change the maintainer label to `maintainers@projectcalico.org`

Reply:
Oh I see, it's because of the multistage build.  Gotcha.  I'm happy to go with the status quo for this, but my personal preference would be to copy in license from local file if that's really all we are using that base image for.  I don't think you should change this unless there was general agreement on that approach though.

Reply:
@robbrockbank  your suggestion is good. Changed this to full scratch. I verified cluster comes up just fine and @lmm verified RH validation is OK with that.